File: C/Exercises/1ª Lista de Exercícios/Exercício01.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n1, n2, resultado;
    printf("Digite o primeiro número: ");
    scanf("%d", &n1);
    printf("Digite o segundo número: ");
    scanf("%d", &n2);
    resultado = n1 + n2;
    printf("A soma de %d + %d é igual a %d.\n", n1, n2, resultado);
    
}

File: Java/1ª Lista de Exercícios (1S)/Exercício01.java
import java.util.Scanner;

public class Exercício01 {
	public static void main(String args[]){
		Scanner scan = new Scanner(System.in);

		System.out.println("Digite o primeiro número: ");
		int n1 = scan.nextInt();
		
		System.out.println("Digite o segundo número: ");
		int n2 = scan.nextInt();
		
		int r = n1 + n2;
		
		System.out.println("A soma dos 2 números é igual a " + r);		

		scan.close();
	}
}
